app/models/proposal.rb

```diff
@@ -0,0 +1,23 @@
+require 'study_management_client'
+
+class Proposal
```

Comment:
We shouldn't really be needing to build a Proposal model inside the Work Orders application. Otherwise we're going to have to duplicate this in every application we need Proposals.

The Study Application client should be building a Proposal model.